webview/src/desktopMain/kotlin/com/multiplatform/webview/web/CefRequestExt.kt

```diff
@@ -0,0 +1,47 @@
+package com.multiplatform.webview.web
+
+import com.multiplatform.webview.setting.WebSettings
+import org.cef.browser.CefBrowser
+import org.cef.browser.CefFrame
+import org.cef.browser.CefRequestContext
+import org.cef.handler.CefRequestContextHandlerAdapter
+import org.cef.handler.CefResourceRequestHandler
+import org.cef.handler.CefResourceRequestHandlerAdapter
+import org.cef.misc.BoolRef
+import org.cef.network.CefRequest
+
+fun createModifiedRequestContext(
```

Comment:
Please change the visibility to `internal`.

We want to expose dependency classes as little as possible since we don't `api` them.

Reply:
Done 👍